src/features/wallet/send/SendView.tsx

```diff
@@ -142,7 +233,97 @@ const SendView = ({ scanResult }: { scanResult?: QrScanResult }) => {
     triggerNavHaptic()
   }
 
-  const constructTxn = async (): Promise<string> => {
+  const handleSellerTransfer = async (): Promise<undefined> => {
```

Comment:
is `Promise<undefined>` required here? could probably leave it off and it'd be inferred from being an async function

Reply:
Good point, removed it